src/main/java/no/nav/pto/veilarbportefolje/database/Table.java

```diff
@@ -107,4 +107,16 @@
         public static final String ANSVARLIG_VEILEDER_NAVN = "ANSVARLIG_VEILEDER_NAVN";
     }
 
+    public static final class FEILET_KAFKA_MELDING {
```

Comment:
Brukes tabellen `FEILET_KAFKA_MELDING` til noe? Finner bare referanse i `KafkaStats`, men den burde kanskje slå opp i `KAFKA_CONSUMER_RECORD` i stede, hvis det er for stats med det nye common biblioteket

Reply:
Yep, that one is for stats, which I need to finish. I need to check if we need `FEILET_KAFKA_MELDING` table

Reply:
Fixed, thanks for the comment